main.go

```diff
@@ -100,6 +101,11 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	if env.MockMode {
+		api.SetMockMode()
+	}
```

Comment:
We don't have to check `testServerConnections` below here now and maybe we should change the logging as we don't do what we log